app/views/state_file/questions/nj_retirement_income_source/edit.html.erb

```diff
@@ -0,0 +1,41 @@
+<%
+    title = t(".title" )
+    content_for :page_title, title
+%>
+
+<% content_for :card do %>
+  <h1 class="h2"><%= title %></h1>
+  <h2 class="h3"><%= t(".subtitle") %></h2>
```

Comment:
[pebble] Does it make sense to use an h2 here? It's not really organizing content. Maybe an [hgroup](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hgroup)?  I'm not sure whether assistive tech will do anything with that but it's what the element is intended for

Reply:
I like the idea of the `hgroup`. It felt weird for it to be a `p` given that it is both visually & semantically distinct from the three `p` tags that follow it for the 1099-R details.